### ISSUE TITLE
Send an e-mail to collaborators when they're added to an existing ticket

### ISSUE THREAD

**Issue by GabrielMajeri** [UniBuc-DITC/osTicket#4]
Right now, when we add a collaborator to an existing ticket, they don't receive any sort of notification e-mail telling them this. They'll receive e-mails only after new messages are added.

We should try to configure or update the app so that it generates and sends an informational message when you're added as a collaborator to a new ticket.
